gk/bpf.c

```diff
@@ -343,13 +343,13 @@ __bpf_jit_if_possible(struct rte_bpf *bpf, rte_bpf_jitted_func_t *ret_f,
 
 	int rc = rte_bpf_get_jit(bpf, &jit);
 	if (unlikely(rc != 0)) {
-		GK_LOG(ERR, "%s() failed to get JIT program %s at index %u, error code: %i\n",
-			__func__, name, index, rc);
+		B_LOG(ERR, "%s() failed to get JIT program %s at index %u, error code: %i\n",
+		      __func__, name, index, rc);
```

Comment:
I may be wrong, so please correct me, but DPDK's coding style encourages parenthesis alignment but does not require it. I'm bringing this up because playing with only TABs is easier than mixing up spaces in these statements. We have been using name alignment for `struct`s as the coding style recommends, but spaces are a lot easier to maintain in that case since we don't edit `struct`s that often.